fuel-core/src/schema/block.rs

```diff
@@ -175,3 +178,53 @@ impl BlockQuery {
         .await
     }
 }
+
+#[derive(Default)]
+pub struct BlockMutation;
+
+#[Object]
+impl BlockMutation {
+    async fn advance_block(
+        &self,
+        ctx: &Context<'_>,
+        advance_by: Option<U64>,
```

Comment:
Same here: the `Option` wrapper just adds complexity for little gain IMO (with the unwrap eg)

Reply:
My thinking with this was the someone could call `mine()` if they just want to advance, then have the option to specify an amount `mine(5)`, with the thinking that it could then be a optional arg. I get now though with Rust that style of design isn't supported and would have to be `mine(None)`, will change though